tic_tac_toe/game.py

```diff
@@ -63,22 +83,50 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    if board[0][0] == player and board[0][1] == player and board[0][2] == player:
```

Comment:
You should be passing sets of coordinates to your `_is_winning_combination` helper function instead of doing all your checks in here.